src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -926,6 +926,20 @@ public void deleteMetadata(String templateName, String scope) {
                 upload(this.getAPI(), this.getID(), inputStream, url, fileName, fileSize);
     }
 
+    /**
+     * Creates a new Metadata Cascade Policy on a folder.
+     *
+     * @param scope         the scope of the metadata cascade policy.
+     * @param templateKey   the key of the template.
+     * @return  information about the Metadata Cascade Policy.
+     */
+    public BoxMetadataCascadePolicy.Info createCascadePolicyOnFolder(String scope, String templateKey) {
```

Comment:
The `OnFolder` part of this method name is redundant — this is a method on `BoxFolder`, after all